.github/workflows/new_updated_issue.yaml

```diff
@@ -1,5 +1,9 @@
 name: Add new and updated issues to Needs Review
 
+env:
```

Comment:
remove this?

Reply:
we still use it for determining which column to move to (lines 21 and 28). I will remove it from the updated_pull_request file however :)

Reply:
updated below